homeassistant/components/binary_sensor/modbus.py

```diff
@@ -16,11 +16,13 @@
 _LOGGER = logging.getLogger(__name__)
 DEPENDENCIES = ['modbus']
 
+CONF_HUB = 'hub'
```

Comment:
Maybe we should define this in the modbus component instead, and import it where needed.

Reply:
Yes, that seems better. Done.